src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DbException;
+
+public class ConnectionUtil {
+    private static final Properties dbProperties = new Properties();
```

Comment:
```suggestion
    private static final Properties DB_PROPERTIES = new Properties();
```
constant naming convention